docs/content-services/components/search-date-range-advanced-tabbed.component.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: Search date range advanced component
+Added: v6.1.0
+Status: Active
+Last reviewed: 2023-06-13
```

Comment:
Also the date should be the day the PR has been created

Reply:
Updated Last Reviewed date to 2023-06-28